tests/conftest.py

```diff
@@ -1,10 +1,24 @@
 import asyncio
 import csv
 import json
+import ssl
+import sys
+from logging import warning
 from pathlib import Path
 from uuid import uuid4
 
 import pytest
+
+try:
+    import truststore
+
+    has_truststore = True
+except ImportError:
+    if sys.version_info > (3, 10):
+        warning(
+            "truststore is not installed, SSL verification will not work. run pip install truststore"
```

Comment:
Instead of this warning let's add it as a dev requirement with a python >= 3.10 marker

Reply:
i kept warning in just in case someone did not sync if you insist on removing it i do not mind

Reply:
It should be impossible to hit the warning, but it also isn't hurting anything so it's fine to leave it.